File: app/src/main/java/com/example/dev/hazikura/fragment/Household/HouseholdFragment.java
package com.example.dev.hazikura.fragment.Household;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentStatePagerAdapter;
import android.support.v4.view.ViewPager;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import com.example.dev.hazikura.R;


/**
     * Created by dev on 2018/03/12.
     */
public class HouseholdFragment extends Fragment {

    ViewPager viewPager;
    ViewPagerAdapter viewPagerAdapter;
    public HouseholdFragment() {
    }

    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState)
    {
        View view = inflater.inflate(R.layout.fragment_household, container, false);

        viewPager = (ViewPager) view.findViewById(R.id.viewpager);
        viewPagerAdapter = new ViewPagerAdapter(getFragmentManager());
        viewPager.setAdapter(viewPagerAdapter);
        return view;
    }

    public static class ViewPagerAdapter extends FragmentStatePagerAdapter {
        private static final int NUM_ITEMS = 2;

        public ViewPagerAdapter(FragmentManager fm) {
            super(fm);
        }

        @Override
        public int getCount() {
            return NUM_ITEMS;
        }

        @Override
        public Fragment getItem(int position) {
            if (position == 0) {
                return InputFragment.newInstance();
            } else {
                return OutputFragment.newInstance();
            }
        }

        @Override
        public CharSequence getPageTitle(int position){
            if (position == 0){
                return "収入";
            }
            else {
                return "支出";
            }
        }
    }

}

File: app/src/main/java/com/example/dev/hazikura/fragment/household/HouseholdFragment.kt
package com.example.dev.hazikura.fragment.household

import android.os.Bundle
import android.support.v4.app.Fragment
import android.support.v4.app.FragmentManager
import android.support.v4.app.FragmentStatePagerAdapter
import android.support.v4.view.ViewPager
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import com.example.dev.hazikura.R


/**
 * Created by yokoro
 */

class HouseholdFragment : Fragment() {

    private lateinit var viewPager: ViewPager
    private lateinit var viewPagerAdapter: ViewPagerAdapter

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        val view = inflater.inflate(R.layout.fragment_household, container, false)

        viewPager = view.findViewById<View>(R.id.viewpager) as ViewPager
        viewPagerAdapter = ViewPagerAdapter(fragmentManager!!)
        viewPager.adapter = viewPagerAdapter
        return view
    }

    class ViewPagerAdapter(fm: FragmentManager) : FragmentStatePagerAdapter(fm) {

        override fun getCount(): Int {
            return NUM_ITEMS
        }

        override fun getItem(position: Int): Fragment {
            return if (position == 0) {
                InputFragment.newInstance()
            } else {
                OutputFragment.newInstance()
            }
        }

        override fun getPageTitle(position: Int): CharSequence {
            return if (position == 0) {
                "収入"
            } else {
                "支出"
            }
        }

        companion object {
            private const val NUM_ITEMS = 2
        }
    }

}

